pkg/uploader/kopia/snapshot.go

```diff
@@ -212,19 +220,24 @@ func SnapshotSource(
 		return "", 0, errors.Wrapf(err, "Failed to flush kopia repository")
 	}
 	log.Infof("Created snapshot with root %v and ID %v in %v", manifest.RootObjectID(), manifest.ID, time.Since(snapshotStartTime).Truncate(time.Second))
-	return reportSnapshotStatus(manifest)
+	return reportSnapshotStatus(manifest, policyTree)
 }
 
-func reportSnapshotStatus(manifest *snapshot.Manifest) (string, int64, error) {
+func reportSnapshotStatus(manifest *snapshot.Manifest, policyTree *policy.Tree) (string, int64, error) {
 	manifestID := manifest.ID
 	snapSize := manifest.Stats.TotalFileSize
 
+	var ErrUnknown = "unknown or unsupported entry type"
+
 	var errs []string
 	if ds := manifest.RootEntry.DirSummary; ds != nil {
 		for _, ent := range ds.FailedEntries {
-			errs = append(errs, ent.Error)
+			if !(*policyTree.DefinedPolicy().ErrorHandlingPolicy.IgnoreUnknownTypes == true && strings.Contains(ent.Error, ErrUnknown)) {
```

Comment:
1. It would be better to check if `policyTree.DefinedPolicy()` is nil, b/c nil is a valid return value of `policyTree.DefinedPolicy()`:
https://github.com/kopia/kopia/blob/master/snapshot/policy/policy_tree.go#L89
1. It would be better to reuse the error fs.ErrUnknown:
https://github.com/kopia/kopia/blob/master/fs/entry.go#L16

Reply:
modified

Reply:
thanks!